src/components/textfield/textfield.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import styled from 'styled-components';
+import { TextfieldLabel } from './../typography/typography';
+
+const StyledLabel = styled(TextfieldLabel)`
+  display: block;
+  margin-bottom: 4px;
+
+  &.error {
+    font-weight: 700;
+    color: #ff0d35;
+  }
+`;
+
+interface Textfield {
+  errorMessage: string;
+  label: string;
+}
+
+const Textfield = ({ label, errorMessage }) => {
+  const StyledInputText = styled.input.attrs({
+    type: 'input',
+  })`
+    width: 320px;
+    height: 48px;
+
+    padding-left: 16px;
+    padding-right: 16px;
+
+    font-size: 16px;
+    line-height: 150%;
+    letter-spacing: -0.01em;
```

Comment:
This is `Text R` from Figma, so please do the same here as suggested for `Text XS` for the label. Add a new (local) typography setting in this file
```ts
const textR = css`
  // ...
`;
```

and use this style in here, like so:
```diff
-    font-size: 16px;
-    line-height: 150%;
-    letter-spacing: -0.01em;
+    ${textR}
```